### PR TITLE
Support for Falcon Dashboard

### PR DESCRIPTION
**Pull request by EliyaFishman**
Follow path was changed to get the X and Y values of the path.
Use [the modified falcon dashboard]( https://github.com/EliyaFishman/FalconDashboard/releases) to check it.

### CODE REVIEW

File: src/main/java/frc/robot/motion_profiling/FollowPath.java [hunk 1]
```diff
@@ -1,21 +1,60 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/FollowPath.java]: remove

File: src/main/java/frc/robot/motion_profiling/FollowPath.java [hunk 2]
```diff
@@ -1,21 +1,60 @@
...
+import static edu.wpi.first.wpilibj.util.ErrorMessages.requireNonNullParam;
 import static frc.robot.Robot.drivetrain;
 import static frc.robot.Robot.robotConstants;
 
+
+/**
+ * A command that uses a RAMSETE controller ({@link RamseteController}) to follow a trajectory
+ * {@link Trajectory} with a differential drive.
+ *
+ * <p>The command handles trajectory-following, PID calculations, and feedforwards internally.  This
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/FollowPath.java]: internally.  This 
too many spaces

File: src/main/java/frc/robot/motion_profiling/FollowPath.java [hunk 3]
```diff
@@ -59,37 +87,180 @@ public FollowPath(Path path, boolean isTuning) {
...
+     * @param trajectory            The trajectory to follow.
+     * @param pose                  A function that supplies the robot pose - use one of
+     *                              the odometry classes to provide this.
+     * @param follower              The RAMSETE follower used to follow the trajectory.
+     * @param kinematics            The kinematics for the robot drivetrain.
+     * @param outputMetersPerSecond A function that consumes the computed left and right
+     *                              wheel speeds.
+     * @param requirements          The subsystems to require.
+     */
+    private FollowPath(Trajectory trajectory,
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/motion_profiling/FollowPath.java]: remove... make the code as minimal as you can...

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
make the code as minimal as you can...

**Comment by EliyaFishman** [comment]
[Use Falcon Dashboard For motion profiling](https://trello.com/c/5XGl8OzU/136-use-falcon-dashboard-for-motion-profiling)

**Review by HilelAmiel — APPROVED** [review]

**Pull request merged**